docs/platform-services/automation-service/app-central/integrations/imap.md

```diff
@@ -158,7 +158,7 @@ The final playbook looks like this:<br/><img src={useBaseUrl('/img/platform-serv
 
 * **IP**: ([0-9]+(?:\.[0-9]+){3})
 * **Email**: [\w\.-]+@[\w\.-]+
-* **URL**: (?i)(?:(?:https?|ftp):\/\/|www\.|ftp\.)(?:\([-A-Z0-9+&@#\/%=~\_|$?!:,.]\*\)|[-A-Z0-9+&@#\/%=~\_|$?!:,.])\*(?:\([-A-Z0-9+&@#\/%=~\_|$?!:,.]\*\)|[A-Z0-9+&@#\/%=~\_|$])
+* **URL**: `(?i)(?:(?:https?|ftp):\/\/|www\.|ftp\.)(?:\([-A-Z0-9+&@#\/%=~\_|$?!:,.]\*\)|[-A-Z0-9+&@#\/%=~\_|$?!:,.])\*(?:\([-A-Z0-9+&@#\/%=~\_|$?!:,.]\*\)|[A-Z0-9+&@#\/%=~\_|$])`
```

Comment:
Hi @jpipkin1 , with this upgrade, it's now stricter with some formatting. This regex query was breaking the Docs build, so I had to add backticks. Is this ok with you?

Reply:
Of course, @kimsauce! I didn't write those anyway. Whatever you have to do to make it work is okay

Reply:
oh ok thank you!